pandas/core/series.py

```diff
@@ -2880,6 +2880,7 @@ def sort_index(
         kind="quicksort",
         na_position="last",
         sort_remaining=True,
+        ignore_index=False,
```

Comment:
can you add type annotations when adding kwargs

Reply:
added!